knoten/vis.py

```diff
@@ -133,10 +133,174 @@ def point_info(cube_path, x, y, point_type, allow_outside=False):
     return pvlres
 
 
-def reprojection_diff(isd, cube, nx=4, ny=8):
+def plot_diff(data, title='diff plot', colx='x', coly='y', coldx='diffx', coldy='diffy', colmag='magnitude', width=500, height=500):
+    import matplotlib.cm as cm
+    from matplotlib.colors import Normalize
+
+    fig = make_subplots(rows=2, cols=2, column_widths=[0.9, .1], row_width=[.9, .1],
+                        shared_xaxes=True, shared_yaxes=True, horizontal_spacing = 0.01, vertical_spacing = 0.01)
+
+    arrow_scale = (data[coly].iloc[1]-data[colx].iloc[0]+data[colx].iloc[1]-data[colx].iloc[0])/5
```

Comment:
should this be `coly[1] - coly[0] + colx[1] - colx[0]`?

Reply:
Is this variable even used? scale looks hard coded to 1

Reply:
It's no longer used, it was before I decided to leave the quiver scale to one so the length is one to one with the pixel offset.